main.rb

```diff
@@ -31,6 +22,50 @@ def list_people
     puts
   end
 
+  def list_rentals_for_person
+    print 'ID of person: '
+    id_of_person = gets.chomp.to_i
+    puts 'Rentals:'
+    @rentals.each do |rent|
+      puts "Date: #{rent.date}, Book #{rent.book.title} by #{rent.book.author}" if rent.person.id == id_of_person
+    end
+    puts
+  end
+end
+
+class CreateClass
+  def initialize(list_handler)
+    @list_handler = list_handler
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    b = Book.new(title, author)
+    @list_handler.books.push(b)
+    puts 'Book created successfully'
+    puts
+  end
```

Comment:
- [ ] great job on cleaning up the main.rb file 👍 however the `create_book` method is not in line with the SOLID principle in particular the single responsibility rule. please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) provided by Microverse for more information.